src/humanize/time.py

```diff
@@ -210,3 +233,253 @@ def naturaldate(value):
     if delta.days >= 5 * 365 / 12:
         return naturalday(value, "%b %d %Y")
     return naturalday(value)
+
+
+def _quotient_and_remainer(value, divisor, unit, minimum_unit, suppress):
```

Comment:
Typo, remainer -> [remainder](https://en.wikipedia.org/wiki/Remainder):
```suggestion
def _quotient_and_remainder(value, divisor, unit, minimum_unit, suppress):
```